### ISSUE TITLE
show labelled pixels and print verbose to console

### ISSUE THREAD

**Issue by ruifanp** [matterport/Mask_RCNN#1566]
New to this technique and deep learning in general.

After training my dataset and running a prediction on a new image, how can I retrieve pixel positions of the region identified by the mask in the image?

Another question is when training my data, how can I make it such that only 1 line is printed to console per epoch? I know that in keras there is a model.fit() option where you can specify the verbose level but I don't see that function in the utils here.
